packages/cli/index.js

```diff
@@ -152,6 +153,9 @@ program
     const changeHandler = (filePath) => {
       logger.info(`[${new Date().toLocaleTimeString()}] Reload for file change: ${filePath}`);
       Promise.resolve('').then(() => {
+        if (filePath.replace(/^.*[\\/]/, '') === SITE_CONFIG_NAME) {
```

Comment:
Did you intend to get the filename from the changed file's path? Maybe can do it with `path.basename()`

Reply:
Yup this is intended to get the filename and `path.basename` will be a better approach. Thanks for the catch! :)